seqfold/utils.go

```diff
@@ -0,0 +1,67 @@
+package seqfold
+
+import (
+	"math"
+
+	"golang.org/x/exp/constraints"
+)
+
+func max[T constraints.Ordered](a, b T) T {
+	if a > b {
+		return a
+	}
+	return b
+}
+
+func min[T constraints.Ordered](a, b T) T {
+	if a < b {
+		return a
+	}
+	return b
+}
+
+func abs(x int) int {
+	if x < 0 {
+		return -x
+	}
+	return x
+}
+
+func Reverse(s string) string {
```

Comment:
Know it's silly but there should be a reverse function in `transform` that you can call via `transform.Reverse(<string>)`